src/components/scenes/newAccount/NewAccountUsernameScene.tsx

```diff
@@ -75,7 +75,7 @@ export const NewAccountUsernameScene = ({ branding }: Props) => {
     dispatch(maybeRouteComplete({ type: 'NEW_ACCOUNT_WELCOME' }))
   })
   const handleNext = useHandler(async () => {
-    dispatch(completeUsername(username))
+    if (!isNextDisabled) dispatch(completeUsername(username))
```

Comment:
Why would the handler ever activate if the isNextDisabled determines if the next button is enabled/disabled? This seems like a noop?